src/StreamdeckTypes/Received/Feedback/Misc/TextTypes.ts

```diff
@@ -0,0 +1,30 @@
+export enum FontFamily {
```

Comment:
There's probably a better place for this file.

Reply:
it's fine like this. might be usable for the `TitleParametersDidChangeEvent`, but can be moved later